kafka-sample/c#/README.md

```diff
@@ -17,46 +18,33 @@ Package Links:
 
 Refer to [c#.proj](c%23.csproj) for version details and if you want to update versions.
 
-On linux systems, the client expects the hosts' ca-certificate at `/etc/pki/tls/ca-bundle.crt` (fedora/redhat defaults).
-If your systems default location differs from that (e.g. Debian/Ubuntu), please make sure to copy or symlink the
-certificate.
+Our Kafka server uses Let’s Encrypt CA for SSL/TLS certificates. Confluent.Kafka library use the operating system's 
+default trusted root CA certificates for secure connections. Please ensure that your system has the Let's Encrypt root 
+certificates installed. These certificates are usually included in the system's trusted root store by default. If not, 
+please install them manually.
 
-Example for Debian/Ubuntu distributions:
+For Debian/Ubuntu distributions, the system CA certificates can be updated as:
 ```bash
-mkdir -p /etc/pki/tls/certs
-ln -s /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+sudo update-ca-certificates
 ```
 
 How to use
 ----------
 
-In order to use the sample please change the following parameters. Those parameters should previously send to you.
+In order to use the sample, please substitute the following parameters with those you have received via secure email.
```

Comment:
I propose: 

To use the sample, please change the following parameters, which should have been sent to you beforehand.

Reply:
Implemented as suggested.